src/mixins/filing-template-mixin.ts

```diff
@@ -315,6 +316,15 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
    * @returns the change filing body
    */
   buildChangeFiling (isDraft: boolean): ChangeFirmIF {
+    let parties = this.getPeopleAndRoles
+
+    // if filing and paying, filter out removed entities and omit the 'action' properties
+    if (!isDraft) {
+      // Filter out parties actions
+      parties = parties.filter(x => x.action !== ActionTypes.REMOVED)
+        .map((x) => { const { action, ...rest } = x; return rest })
+    }
```

Comment:
You probably don't need to do this. The API should ignore fields it doesn't care about.

Nice map though :)

Reply:
Just following what we've done in the past with the actions. Will do a little research and remove if possible